index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="new project for microverse">
+    <link rel="stylesheet" href="/style/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Ubuntu:wght@300&display=swap"
+        rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@200;400&family=Ubuntu:wght@300&display=swap"
+        rel="stylesheet">
+    <title>My portfolio project</title>
+</head>
+
+<body>
+    <header id="navbar">
+        <div id="menubar">
+            <img id="shapes" src="/style/img/shapes.svg" alt="shapes">
+            <img id="menu" src="/style/img/menu.svg" alt="menu-button">
+        </div>
+    </header>
+
+    <main id="mainCont">
+        <h1 id="name">
+            Damian M. Dib Bustros
+        </h1>
+        <br>
+        <p id="introduction">
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my
+            work. <br> If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+
+        <div class="buttons">
+            <a href="#"class="submit-button">Start Collaboration</a>
+        </div>
+
+    </main>
```

Comment:
- [ ] Please make this a `section` tag, not `main`. This is a project requirement. 📖  It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.